spec/errors_savior/support/helpers/parsed_response.rb

```diff
@@ -0,0 +1,7 @@
+module Response
+  module JSONParser
+    def response_body
+      ActiveSupport::JSON.decode(response.body) if response.present? && response.body.present?
```

Comment:
Mmmm I think you can rewrite this:
```
      ActiveSupport::JSON.decode(response.body) if response&.body&.present?
```

Reply:
Sure! I've copied this file from bootstrap, but I like this change, thanks!